MekHQ/src/mekhq/campaign/parts/MissingProtomekSensor.java

```diff
@@ -112,7 +111,7 @@ public void fix() {
     @Override
     public boolean isAcceptableReplacement(Part part, boolean refit) {
         return part instanceof ProtomekSensor
-                && getUnitTonnage() == ((ProtomekSensor)part).getUnitTonnage();
+                && getUnitTonnage() == part.getUnitTonnage();
```

Comment:
I assume ProtomekSensor doesn't have an implementation of getUnitTonnage()?

Reply:
Shouldn't need the cast to call it if it overrides the base method. (I don't think)

Reply:
@sixlettervariables is correct, you don't need to cast for an overridden method.